dlmalloc/src/malloc.c

```diff
@@ -5228,22 +5228,10 @@ static void try_init_allocator(void) {
   char *base = &__heap_base;
   // Try to use the linker pseudo-symbol `__heap_end` for the initial size of
   // the heap, but if that's not defined due to LLVM being too old perhaps then
```

Comment:
can you remove `perhaps` here.

How about ".. but if that's not defined (due to LLVM being too old) ..."

Reply:
done